### ISSUE TITLE
Training details for released GRM series

### ISSUE THREAD

**Issue by 1nit-lit** [YangRui2015/Generalizable-Reward-Model#7]
Thanks for the excellent work and open source code!

It's great to have the released models. Are the GRM series (2B, 3B, 8B) on HuggingFace all trained on the entire dataset of "Preference-700K" with full-parameter, and details stated in Appendix B?

**Comment by YangRui2015** [comment]
You can find the used dataset in each model card on huggingface (-sftreg models generally use "Preference-700K" and -ft models use Skywork-80K-v0.2). The released models are trained with full-parameter as in Appendix B. The only different hyperparameters is the learning rate. Specifically, 2B and 3B models use slightly larger learning rates such as 5e-6. In addition, some intermediate checkpoints can have better performance.

![Image](https://github.com/user-attachments/assets/1b977aa0-bfda-4924-947e-b97aa9bffcdf)

**Comment by 1nit-lit** [comment]
Thank you for the details!
